addon/mixins/in-viewport.js

```diff
@@ -59,7 +59,9 @@ export default Ember.Mixin.create({
   },
 
   _setupElement: on('didInsertElement', function() {
-    if (!canUseDOM) {
+    const noSetup = !get(this, 'viewportSpy') && get(this, 'noSetup');
```

Comment:
@kellyselden Agreed! @poteto should I rebase and update according to kellyselden's suggestion?